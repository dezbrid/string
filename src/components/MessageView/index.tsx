import React from 'react';
import {View, Text} from 'react-native';

import styles from './styles';

interface Props {
  message: string;
}
function MessageView({message}: Props) {
  return (
    <View style={styles.container}>
      <Text>{message}</Text>
    </View>
  );
}
export default MessageView;
