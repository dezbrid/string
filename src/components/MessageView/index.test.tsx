import 'react-native';
import React from 'react';
import {render} from '@testing-library/react-native';

import MessageView from './index';

describe('<MessageView />', () => {
  test('<MessageView /> snapshot', () => {
    const tree = render(<MessageView message="test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('should to see message Test', () => {
    const {findByText} = render(<MessageView message="Test" />);
    expect(findByText('Test')).toBeTruthy();
  });
});
