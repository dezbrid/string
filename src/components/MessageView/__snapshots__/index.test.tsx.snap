// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MessageView /> <MessageView /> snapshot 1`] = `
<View
  style={
    Object {
      "margin": 20,
    }
  }
>
  <Text>
    test
  </Text>
</View>
`;
